Plugins/JiraCommitTipPlugin/JiraComminTipPlugin.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Windows.Forms;
+using Atlassian.Jira;
+using GitUI.CommandsDialogs;
+using GitUIPluginInterfaces;
+using JiraCommitPlugin;
+using ResourceManager;
+
+namespace JiraComminTipPlugin
+{
+    public class JiraComminPlugin : GitPluginBase, IGitPluginForRepository
+    {
+        private const string description = "Jira Commit Tip";
+        private Jira jira;
+        private JiraNamedEntity filter;
+        private readonly StringSetting url = new StringSetting("Jira URL", @"https://jira.atlassian.com");
+        private readonly StringSetting user = new StringSetting("Jira user", string.Empty);
+        private readonly PasswordSetting password = new PasswordSetting("Jira password");
+        private readonly StringSetting filterName = new StringSetting("Filter name", "[Filter should be in your favorites]");
+        private FormCommit commitForm;
+
+        public JiraComminPlugin()
+        {
+            Description = description;
+        }
+
+        public override bool Execute(GitUIBaseEventArgs gitUiCommands)
```

Comment:
Is there a way to avoid calling this method by hiding plugin from direct execution list?